docs/snippets/common/my-component-story-use-globaltype.mdx.mdx

```diff
@@ -8,14 +8,14 @@ export const getCaptionForLocale = (locale) => {
     case 'kr': return '안녕하세요!';
     case 'zh': return '你好!';
     default:
-      return 'Hello!',
+      return 'Hello!';
   }
 }
 
 <Story name="StoryWithLocale">
   {(args, { globals: { locale } }) => {
     const caption = getCaptionForLocale(locale);
-    return <>{caption}</>;
+    return `<p>${caption}</p>`;
   }}
 </Story>
```

Comment:
@nicoleoliveira we could leave the fragment in this case.

Reply:
When I left `<> {caption} </>` it generated an error when compiling the storybook. I thought it was a specific syntax, I didn't understand that it could be modified. So I thought it best to put an html tag as an example to make it more intuitive. What do you think? :)

Reply:
@nicoleoliveira let me run the code once again and i'll get back to you.

Stay safe

Reply:
@nicoleoliveira it checks out. Here's a sample [repo](https://github.com/jonniebigodes/storybook-react-for-issue-13593/tree/test_nicole_toolbar_example_code) with the code and a [deployed](https://gallant-agnesi-5aff3c.netlify.app/?path=/story/task--story-with-locale) version. 

Testing done:
- Development mode with `npm run storybook` and production mode with `npm run build-storybook && npm run serve-prod" (`serve-prod` is a script that i've included to emulate a production server running the statically built storybook). 
- Bumped versions back  and all check out with the test above. 

If you could emulate the error you've mentioned, please feel free to open an issue and if possible add a reproduction so that we could take a look at it. In terms of this pull request we're good with leaving in the fragment. But nonetheless we're truly thankful for catching that typo and fix it.

Let me know and we'll get this merged.

Stay safe

Reply:
Hi @jonniebigodes … I appreciate your attention. I think you tested in .js file, but I found the problem in the .mdx. Let me try to be clearly:

![Captura de tela de 2021-01-18 09-12-29](https://user-images.githubusercontent.com/17319734/104914058-5195a100-596d-11eb-8c27-425e32808f4a.png)

That’s the sample code in the page of the story book. In the line where we can read “return <>{caption}</>;” throws an error saying that an html tag was expected.

I was trying to say that maybe, should be clear if the docs use some html tag there to just run the code and work. As example:

return `<p>${caption}</p>`;

I’m working with web-components and using mdx. If using the tags like: “<></>” it should work, so a real error is happening.

There is my [repo link as an example](https://github.com/nicoleoliveira/storybook-web-components/tree/example-storybook).
Execute `npm run storybook` to test.

Reply:
Sorry for the fumble fingering. I forgot to commit the mdx portion :see_no_evil: . I've just pushed the updated config to the repo. But i'll keep looking into this, i'm going to clone the repo you've supplied (thank you very much for that) and see what's going on and let you know of my findings and we'll move from there.

Sounds good?

Reply:
Sound good! Appreciate it. :)